package.json

```diff
@@ -12,6 +12,7 @@
     "@patternfly/react-icons": "3.14.23",
     "bootstrap": "3.4.1",
     "core-js": "3.4.0",
+    "core-js-compat": "3.4.5",
```

Comment:
This is obsolete now, at least cockpit's tests run fine again. 3.4.7 now seems to be generally available. That said, don't let that block this PR, as that's a bit urgent.